Filed/AFH.md

```diff
@@ -3,7 +3,7 @@
 ***Description***      :: `= this.link`
 ***Author*** :: [[FAA]]
 
-Chapters:
+**Chapters**:
```

Comment:
Small style change seems ok to group in this PR, although I didn't make the same change in the similarly structured PHAK.md file.